src/main/java/org/springframework/data/couchbase/core/query/StringQuery.java

```diff
@@ -21,16 +21,12 @@
 import org.springframework.data.couchbase.repository.query.StringBasedN1qlQueryParser;
 
 /**
- * @author Michael Reiche
- *
- * Query created from the string in @Query annotation in the repository interface.
- *
- *      @Query("#{#n1ql.selectEntity} where firstname = $1 and lastname = $2")
- * 	    List<User>  getByFirstnameAndLastname(String firstname, String lastname);
+ * Query created from the string in @Query annotation in the repository interface. @Query("#{#n1ql.selectEntity} where
```

Comment:
What's the Spring convention for including examples in Javadoc? Should the code snipped be enclosed in "pre" tags?

Reply:
Fixed. enclosed in "pre"